docker/test/installer/linux.Dockerfile

```
@@ -72,6 +72,7 @@ RUN . /etc/profile && nvq++ --help
 
 ## [ADD tools for validation]
 ADD scripts/validate_container.sh /home/cudaq/validate.sh
+ADD scripts/configure_build.sh /home/cudaq/build_environment.sh
```

I would recommend keeping the filenames the same to make these scripts a bit easier to follow. If you accept this change, you'll probably need to update your new changes in `.github/workflows/prebuilt_binaries.yml`, too.
```suggestion
ADD scripts/configure_build.sh /home/cudaq/configure_build.sh
```

---

done